.github/workflows/move-to-polka.yml

```diff
@@ -58,32 +58,22 @@ jobs:
         if: matrix.os == 'macos-latest'
         run: |
           brew install llvm@19 lld
-      
+
       - name: Check ld.ldd version
         run: |
           which ld.lld
           ld.lld --version
 
       - name: Install Rust
-        if: matrix.rust != 'nightly'
```

Comment:
is it still useful to have nightly in the matrix?

Reply:
Personally, I think it is always good to test QA on more than one version to check for different types of issues.